File: Ekklesia.Spa/src/app/models/Role.ts
export enum Role {
  MEMBRO = 1,
  LIDER = 2,
  PROFESSOR = 3,
  INDEFINIDO = 4,
}

export const RoleMapping: Record<string | Role, string> = {
  [Role.MEMBRO]: 'Membro',
  [Role.LIDER]: 'Líder',
  [Role.PROFESSOR]: 'Professor(a)',
  [Role.INDEFINIDO]: 'Indefinido(a)',
}

File: Ekklesia.Entities/Enums/Role.cs
using System.ComponentModel;

namespace Ekklesia.Entities.Entities
{
    public enum Role
    {
        [Description("Membro")]
        MEMBRO = 1,
        [Description("Líder")]
        LIDER = 2,
        [Description("Professor(a)")]
        PROFESSOR = 3,
        [Description("Não definido(a)")]
        INDEFINIDO = 4,
    }

}
